templates/index.html

```diff
@@ -1024,6 +1023,24 @@ <h4 class="title"></h4>
         inGooglePlay: 'בGoogle play'
     });
 </script>
+
+<script>
+    // show filter panel on search string click
+    function showFilter(number){
+        switch (number){
+            case 0:
```

Comment:
See comment above about constants.
Or -- just pass the id instead of a number. Wouldn't that be more readable and less code?